omi/sawtooth_omi/handler.py

```diff
@@ -72,11 +72,10 @@ def _get_unique_key(obj, action):
     return key
 
 
-def make_omi_address(obj, action):
+def make_omi_address(name, action):
```

Comment:
Yeah, it's not great. I'll fix that after I get the rest of this functionality written.